### ISSUE TITLE
Selfhosted extension browser error

### ISSUE THREAD

**Issue by ICCTAM** [bitwarden/server#2524]
### Steps To Reproduce

.

### Expected Result

I have a bitwarden self hosted server at home, and I published it, I'm able to run it on my phone normally but whenever I try to login outside my network on a browser extension it gives me the error "username or password incorrect" it's only in the browser extension, if I log in directly in the url it works

### Actual Result

.

### Screenshots or Videos

![image](https://user-images.githubusercontent.com/78036600/210074303-40419425-caf0-40c9-8968-dd184c2ef2b6.png)

### Build Version

Version 2.25.1

### Environment

Self-Hosted

### Environment Details

Docker

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by justindbaur** [comment]
I believe the version `2.25.1` you provided is the version of your browser extension. What is the version of your self hosted Bitwarden installation. We sent out [this deprecation notice](https://bitwarden.com/help/november-deprecation-notice/), I have a feeling your server is pre 1.46.0 and you will need to update.

**Comment by djsmith85** [comment]
Hi @ICCTAM and thank you for your report.

Based on the server version you provided, you are likely not using an official Bitwarden server. All official server releases can be found [here](https://github.com/bitwarden/server/releases) and [here](https://github.com/bitwarden/self-host/releases).

Please seek help from the maintainer of your unofficial Bitwarden server or have a look into migrating to the offical deployment option [here](https://bitwarden.com/help/install-on-premise-linux/) or try out newest [unified beta](https://bitwarden.com/help/install-and-deploy-unified-beta/). 

Kind regards,
Daniel

**Comment by ICCTAM** [comment]
![image](https://user-images.githubusercontent.com/78036600/210077544-d844142b-c5a4-44de-a804-903c796254e8.png)
Just notice that, guess I choose the wrong docker image, it`s not maintained by bitwarden, thanks I`ll deploy a new one!!

**Comment by ICCTAM** [comment]
Is it possible to install it in a different port than 80? I run the installation but it`s inside my portainer server, or if is there a composer script it would be great

**Comment by djsmith85** [comment]
@ICCTAM In our [hosting FAQ](https://bitwarden.com/help/hosting-faqs/#q-how-do-i-use-custom-server-ports) there is a section showing how to configure custom ports.

**Comment by ant0nwax** [comment]
Hi there

I think the deprecated notice was not being sent out by the on premises hosted servers, where to subscribe to get these ultra important notices? does anyone know?

I just debugged 2 hours to find out why browser extensions do not work anymore... thanks all for contributing here...
browser extensions work again after I updated my reverse proxy with yum update -y, the bitwarden server with apt update && apt upgrade -y and  running ./bitwarden.sh updateself and ./bitwarden.sh update TWO times.
